scripts/stateful/generate_upgrade_matrix_v16.py

```diff
@@ -5,7 +5,7 @@
 import re
 
 SKIP_STARTING_VERSIONS = ['v15.0.0-rc0','v15.0.0-rc1','v15.0.0-rc2','v15.0.0-rc3','v15.0.0','v15.1.0','v16.0.0-rc0','v16.0.0-rc1','v16.0.0-rc2','v16.0.0-rc3','v16.0.0']
-SKIP_TARGET_VERSIONS = []
+SKIP_TARGET_VERSIONS = ['v16.0.0-rc0','v16.0.0-rc1']
```

Comment:
```suggestion
SKIP_TARGET_VERSIONS = ['v16.0.0-rc0','v16.0.0-rc1','v16.0.0-rc2']
```
The final release is out now